scripts/validate_string_concatenation.py

```diff
@@ -1,51 +1,39 @@
 #!/usr/bin/env python
 """
-GH #30454
-
-Check where there is a string that needs to be concatenated.
-
-This is necessary after black formating,
-where for example black transforms this:
-
->>> foo = (
-...         "bar "
-...         "baz"
-...     )
-
-into this:
-
->>> foo = ("bar " "baz")
-
-Black is not considering this as an
-issue (see issue https://github.com/psf/black/issues/1051),
-so we are checking it here.
+Unwanted patterns test cases.
```

Comment:
Can you give some context here? Like why this script exists (why this stuff is not in `ci/code_checks.sh` directly like the rest.